src/components/Chip/ChipWithSelect.tsx

```diff
@@ -0,0 +1,73 @@
+import React, {
+  InputHTMLAttributes,
+  OptionHTMLAttributes,
+  useState,
+} from 'react'
+import styled from '@emotion/styled'
+import { ChevronDown } from '../../icons'
+import { Chip, ChipProps } from './Chip'
+
+const Content = styled.span`
+  select {
+    position: absolute;
+    opacity: 0;
+    inset: 0;
+    cursor: pointer;
+  }
+`
+
+interface OptionProps extends OptionHTMLAttributes<HTMLOptionElement> {}
```

Comment:
If we don't add any extra properties, why extend something instead of using the thing that you're using to extend?